src/gui/application.cpp

```diff
@@ -108,6 +108,7 @@ Application::Application(int &argc, char **argv)
     , _userTriggeredConnect(false)
     , _debugMode(false)
     , _backgroundMode(false)
+    , _isQuitting(false)
```

Comment:
IMHO a more safe and efficient way than introducing a new member variable (which resembles a state to be at least be passively aware of and that has to be actively maintained/set in some current or future places) would be to handle this within the specific method. See comment on slotCleanup()